text/RFC-0012-Process-for-Introducing-New-Collectives.md

```diff
@@ -0,0 +1,108 @@
+
+Since the introduction of the Collectives parachain, many groups have expressed interest in forming
+new -- or migrating existing groups into -- on-chain collectives. While adding a new collective is
+relatively simple from a technical standpoint, the Fellowship will need to merge new pallets into
+the Collectives parachain for each new collective. This RFC proposes a means for the network to
+ratify a new collective, thus instructing the Fellowship to instate it in the runtime.
+
+## Motivation
+
+Many groups have expressed interest in representing collectives on-chain. Some of these include:
+
+- Parachain technical fellowship (new)
+- Fellowship(s) for media, education, and evangelism (new)
+- Polkadot Ambassador Program (existing)
+- Anti-Scam Team (existing)
+
+Collectives that form part of the core Polkadot protocol should have a mandate to serve the
+Polkadot network. However, as part of the Polkadot protocol, the Fellowship, in its capacity of
+maintaining system runtimes, will need to include modules and configurations for each collective.
+
+Once a group has developed a value proposition for the Polkadot network, it should have a clear
+path to having its collective accepted on-chain as part of the protocol. Acceptance should direct
+the Fellowship to include the new collective with a given initial configuration into the runtime.
+However, the network, not the Fellowship, should ultimately decide which collectives are in the
+interest of the network.
+
+## Stakeholders
+
+- Polkadot stakeholders who would like to organize on-chain.
+- Technical Fellowship, in its role of maintaining system runtimes.
+
+## Explanation
+
+The group that wishes to operate an on-chain collective should publish the following information:
+
+- Charter, including the collective's mandate and how it benefits Polkadot. This would be similar
+  to the
+  [Fellowship Manifesto](https://github.com/polkadot-fellows/manifesto/blob/0c3df46/manifesto.pdf).
+- Seeding recommendation.
+- Member types, i.e. should members be individuals or organizations.
+- Member management strategy, i.e. how do members join and get promoted, if applicable.
+- How much, if at all, members should get paid in salary.
+- Any special origins this Collective should have outside its self. For example, the Fellowship
+  can whitelist calls for referenda via the `WhitelistOrigin`.
+
+This information could all be in a single document or, for example, a GitHub repository.
+
+After publication, members should seek feedback from the community and Technical Fellowship, and
+make any revisions needed. When the collective believes the proposal is ready, they should bring a
+remark with the text `APPROVE_COLLECTIVE("{collective name}, {commitment}")` to a Root origin
+referendum. The proposer should provide instructions for generating `commitment`. The passing of
+this referendum would be unequivocal direction to the Fellowship that this collective should be
+part of the Polkadot runtime.
```

Comment:
Who is expected to work on introducing an approved collective to the runtime/code base?
Should we clarify it in this RFC?

Reply:
Not exact but similar to: https://github.com/polkadot-fellows/RFCs/pull/2#discussion_r1254675263 

If the Fellowship is being paid by the network and they agree to a process to signal network desire for some code in a runtime that is within the Fellowship GitHub org, then someone in the Fellowship should take responsibility to make sure it gets done. The PR will need approval from high-ranking Fellows even if someone outside the Fellowship implements it.